migrations/2021-12-16-003905_names_to_lowercase/up.sql

```diff
@@ -0,0 +1,37 @@
+-- Your SQL goes here
+
+
+-- Delete person name dupes, keeping the first one
+
+delete
+from person
+where id not in (
+  select min(id)
+  from person
+  group by lower(actor_id)
+);
+
+-- Store the old values so that down.sql can recover them
+alter table person add column old_name varchar(255);
+update person set old_name = name;
+
+alter table person add column old_actor_id varchar(255);
+update person set old_actor_id = actor_id;
+
+alter table person add column old_inbox_url varchar(255);
+update person set old_inbox_url = inbox_url;
```

Comment:
I wouldn't worry about saving these, and it also might mess up some things with diesel if we add columns but don't do it also in the code. Don't worry, before we merge this and run it in production, I will test with our lemmy.ml data to make sure it works correctly. But thanks for thinking of this.